pkg/secrets/passphrase/manager.go

```diff
@@ -186,7 +186,7 @@ func NewPassphaseSecretsManagerFromState(state json.RawMessage) (secrets.Manager
 	// not. Ideally, we would prompt the user for the passphrase at this point but we can't do that in the CLI is in an
 	// update operation, so we should at least error out with an appropriate message here to ensure that the user
 	// understands why the operation fails unexpectedly.
-	if phrase == "" {
+	if !isFound && phrase == "" {
```

Comment:
Would `!isFound` be enough?

Reply:
It *could* be but I am super keen to unblock these users - I can test that out and make it simpler once this P1 is fixed up

Reply:
Ok, tested and you are correct that on;y the check for !isFound is necessary here - we don't care about the "" part

Reply:
Looking at the code, you never return `false` with a non-empty phrase, it seems. So, it looks like no risk to me. I'll leave it to your judgment.